src/frontegg-client/src/client/app_password.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use mz_frontegg_auth::AppPassword;
+use reqwest::Method;
+use serde::{Deserialize, Serialize};
+use uuid::Uuid;
+
+use crate::error::ErrorExtended;
+
+use super::Client;
+
+const APP_PASSWORDS_PATH: [&str; 5] = ["identity", "resources", "users", "api-tokens", "v1"];
+const CREATE_APP_PASSWORDS_PATH: [&str; 6] = [
+    "frontegg",
+    "identity",
+    "resources",
+    "users",
+    "api-tokens",
+    "v1",
+];
```

Comment:
This seems to have the same weirdness as the users paths—one has `frontegg` and the other does not!